forui_hooks/lib/src/select_group_controller_hook.dart

```diff
@@ -0,0 +1,111 @@
+// Flutter imports:
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+// Package imports:
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:forui/forui.dart';
+
+/// Creates a [FRadioSelectGroupController] that allows only one selection to be selected mimicking the behaviour of
+/// radio buttons and is automatically disposed.
+FRadioSelectGroupController<T> useFRadioSelectGroupController<T>({
+  T? value,
+  List<Object?>? keys,
+}) =>
+    use(_RadioControllerHook<T>(value: value));
```

Comment:
Should we pass the keys in too? I think it's missing for some of the other ones too.